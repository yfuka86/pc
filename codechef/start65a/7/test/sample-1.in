2
5
1 2
2 3
1 4
2 5
3 6 8 2 1
2
1 2
4 7
