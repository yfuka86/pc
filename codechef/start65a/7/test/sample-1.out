9 11 11 5 6
7 9
